source/features/mark-merge-commits-in-list.tsx

```diff
@@ -0,0 +1,63 @@
+import './mark-merge-commits-in-list.css';
+import select from 'select-dom';
+import gitmerge from 'octicon/git-merge.svg';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import {getRepoGQL} from '../libs/utils';
+
+const getMergeCommits = async (commits: string[]): Promise<string[]> => {
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+			${commits.map((commit: string) => `
+				${api.escapeKey(commit)}: object(expression: "${commit}") {
+					... on Commit {
+							parents{
+								totalCount
+							}
+						}
+					}
+			`).join('\n')}
+		}
+	`);
+
+	return filterCommits(repository);
```

Comment:
You can inline this function in here. `getMergeCommits` should become `filterMergeCommits`